docs/source/features/parallelisms.rst

```diff
@@ -156,6 +184,40 @@ The NeMo implementation of PP leverages functionalities from Megatron Core. For
 
 For more detailed API usage and configurations related to PP, visit the `Megatron Core Developer Guide <https://docs.nvidia.com/Megatron-Core/developer-guide/latest/api-guide/tensor_parallel.html>`_.
 
+Expert Parallelism
+^^^^^^^^^^^^^^^^^^
+Expert Parallelism (EP) is a type of model parallelism that distributes experts of an MoE across GPUs.
+Unlike other model-parallel techniques, EP is applied to only the expert layers thus does not impact the parallel mapping of the rest of layers.
```

Comment:
Unlike other model-parallel techniques, EP is applied to only the expert layers and does not impact the parallel mapping of the rest of the layers.